hack/gosec.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# Example:  ./hack/gosec.sh
+set -x
+
+if [ "$IS_CONTAINER" != "" ]; then
+  go get github.com/securego/gosec/cmd/gosec
+  gosec -severity medium -confidence medium -exclude G304 ./cmd/... ./data/... ./pkg/... "${@}"
+else
+  podman run --rm \
+    --env IS_CONTAINER=TRUE \
+    --volume "${PWD}:/go/src/github.com/openshift/installer:z" \
+    --workdir /go/src/github.com/openshift/installer \
+    docker.io/openshift/origin-release:golang-1.12 \
```

Comment:
All of our other `hack/` scripts use `golang-1.10`.  I'd rather stick with that if possible, and then bump them all when the repo as a whole moves to 1.12 (or any other Go version).

Reply:
It doesn't work with that image... I only got it to work with 1.12

Reply:
Is using 1.10 still required? I noticed other openshift repos using 1.12